package.json

```diff
@@ -23,6 +23,7 @@
     "@typescript-eslint/parser": "5.21.0",
     "async-retry": "1.2.3",
     "buffer-replace": "1.0.0",
+    "create-svelte": "2.0.0-next.202",
```

Comment:
Is this needed?

Reply:
It's needed as a prerequisite to the cron PR.

Reply:
@tcc-sejohnson I think we should used `npx` instead so it always uses the latest.

See this example for updating Next.js https://github.com/vercel/vercel/blob/5120689bf2b6201ace3f2349c0438f451ff13209/utils/update-next.js#L35

Reply:
@styfle 

We cannot. You can't pass CLI arguments to `npx create svelte@latest`. See https://github.com/vercel/vercel/pull/9030#discussion_r1042641320

The cron script I adapted from the next script does keep it up-to-date, though.